### DISCUSSION TITLE
Welcome to OData.Neo Discussions!

### DISCUSSION THREAD

**Discussion by hassanhabib** [OData/OData.Neo#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaryAnnSelander** [comment]
Great Expectation!  C# Everywhere.

**Comment by Khaos66** [comment]
Hi 👋 
Is this project still pre-alpha or can I use it for evaluation already?

**Comment by Khaos66** [comment]
Ok. It now looks dead already. @hassanhabib

**Comment by jinmaxtor** [comment]
alguna novedad respecto a esto? o debo tomarlo como abandonado?
